src/kinetics/InterfaceKinetics.cpp

```diff
@@ -595,4 +595,30 @@ void InterfaceKinetics::setPhaseStability(const size_t iphase, const int isStabl
     }
 }
 
+double InterfaceKinetics::InterfaceCurrent(int iPhase)
+{
+    int sp = thermo(iPhase).nSpecies();
+    double charge_k[sp];
+    double net_k[sp];
```

Comment:
Again, nitpicky, but something like `sdot_k` would be more consistent with the nomenclature in the literature, which makes it a little easier for an interested reader to comprehend.

Reply:
Agreed, this has been changed.